extensions-contrib/prometheus-emitter/src/main/java/org/apache/druid/emitter/prometheus/PrometheusEmitter.java

```diff
@@ -202,11 +209,35 @@ public void close()
   {
     if (strategy.equals(PrometheusEmitterConfig.Strategy.exporter)) {
       if (server != null) {
-        server.stop();
+        server.close();
       }
     } else {
       exec.shutdownNow();
       flush();
+
+      try {
+        if (config.getWaitForShutdownDelay().getMillis() > 0) {
```

Comment:
It'll also be good to log an `info` log before the wait and delete metrics operation

```suggestion
        if (config.getWaitForShutdownDelay().getMillis() > 0) {
         log.info("Waiting for [%d]ms before deleting metrics from the push gateway.",
                   config.getWaitForShutdownDelay().getMillis());
        
```

Reply:
Done